pkg/controlplane/apiserver/options/validation_test.go

```diff
@@ -86,8 +98,16 @@ func TestValidateAPIPriorityAndFairness(t *testing.T) {
 			if errs := validateAPIPriorityAndFairness(options); len(errs) > 0 {
 				errMessageGot = errs[0].Error()
 			}
-			if !strings.Contains(errMessageGot, test.errShouldContain) {
-				t.Errorf("Expected error message to contain: %q, but got: %q", test.errShouldContain, errMessageGot)
```

Comment:
this test was always broken, `strings.Contains(errMessageGot, "")` is always `true`, it's fixed below.